coderd/notifications/notifications_test.go

```diff
@@ -1086,8 +1086,42 @@ func TestNotificationTemplates_Golden(t *testing.T) {
 				UserUsername: "bobby",
 				Labels: map[string]string{
 					"workspace": "bobby-workspace",
-					"threshold": "90%",
-					"volume":    "/home/coder",
+				},
+				Data: map[string]any{
+					"volumes": []map[string]any{
+						{
+							"path":      "/home/coder",
+							"threshold": "90%",
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "TemplateWorkspaceOutOfDisk",
```

Comment:
Nit: same test name twice.

```suggestion
			name: "TemplateWorkspaceOutOfDisk_MultipleVolumes",
```